core/lib/dal/src/contract_verification_dal.rs

```diff
@@ -313,6 +426,79 @@ impl ContractVerificationDal<'_, '_> {
         Ok(())
     }
 
+    pub async fn save_etherscan_verification_request_id(
```

Comment:
Maybe it makes sense to create `EtherscanVerificationDal`? This file feels a bit overloaded already.

Reply:
Agreed, done.